critiquebrainz/frontend/templates/moderators/moderators.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+
+{% block title %}{{ _('Moderators') }} - CritiqueBrainz{% endblock %}
+
+{% block content %}
+<div>
```

Comment:
This `div` should be indented. Also keep in mind that we use 2 spaces for indentation in templates.